.gitignore

```diff
@@ -3,6 +3,8 @@ yarn-error.log
 node_modules
 .DS_STORE
 .vscode
+tags
+yarn.lock
```

Comment:
why is `yarn.lock` out?

Reply:
ooops sorry, my bad, I added it so it was ignored while grepping. will remove it.